Aliases/libvips

```diff
@@ -0,0 +1 @@
+Formula/vips.rb
```

Comment:
The alias  should point to `../Formula/vips.rb` (note the leading `../`)

Reply:
Oops, should be fixed now.